Assignments/TestFour.java

```diff
@@ -0,0 +1,105 @@
+package assignments.qatechhub;
+
+import org.testng.annotations.Test;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Parameters;
+
+import java.util.Iterator;
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.annotations.AfterClass;
+
+public class TestFour {
+  WebDriver driver;
+  
+  @Parameters({ "Browser" })
+  @BeforeClass
+  public void openEbay(String browser) {
+	  if(browser.equalsIgnoreCase("Chrome"))
+	  {
+		  WebDriverManager.chromedriver().setup();
+		  driver = new ChromeDriver();
+	  }
+	  else if(browser.equalsIgnoreCase("Firefox")) {
+		  WebDriverManager.firefoxdriver().setup();
+		  driver = new FirefoxDriver();
+	  }
+	  else {
+		  System.out.println("Wrong Choice");
+	  }
+	  driver.manage().window().maximize();
+	  driver.get("https://www.ebay.com");
+
+  }
+  
+  @Test(priority = 0)
+  public void search() {
+	  driver.findElement(By.id("gh-ac")).sendKeys("Apple watches");
+	  WebElement drop = driver.findElement(By.id("gh-cat"));
+	  Select select = new Select(drop);
+	  select.selectByVisibleText("Cell Phones & Accessories");
+	  driver.findElement(By.id("gh-btn")).click();
+  }
+
+  @Test(priority = 1)
+  public void result() {
+	  String result = driver.findElement(By.xpath("//h1[contains(text(),'results for ')]")).getText();
+	  System.out.println("Results are: "+result);
+	  System.out.println("---------------------------------------------------------");
```

Comment:
try using logger once.
Because this syso will just print on console ,whereas logger will log all the information in log file